OKTA-352148 .NET > Add content for Blazor in Developer Docs > "Sign i… (#1972)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-spa/handle-callback/blazorwebasm/handle-callback.md

> After completing the configuration in the previous steps, Sign on and Sign out callbacks will be handled automatically by `Microsoft.AspNetCore.Components.WebAssembly.Authentication` library.

Reviewer: ```suggestion
After configuration in the previous steps is complete, Sign on and Sign out callbacks are handled automatically by `Microsoft.AspNetCore.Components.WebAssembly.Authentication` library.
```
Active voice.
